src/main/java/nl/topicus/mssql2monetdb/CopyToolConfig.java

```diff
@@ -128,6 +128,55 @@ public CopyToolConfig(String args[]) throws ConfigException, ConfigurationExcept
 		OptionBuilder.withLongOpt("switch-only");
 		options.addOption(OptionBuilder.create("so"));
 
+		//user
+		OptionBuilder.hasArg(true);
+		OptionBuilder.isRequired(false);
+		OptionBuilder.withDescription("MonetDB username");
+		OptionBuilder.withLongOpt("monetdb-user");
+		options.addOption(OptionBuilder.create("u"));
```

Comment:
Ik twijfel of deze short-options wel handig zijn omdat `u` zowel de monetdb-user maar ook de sqlserver-user zou kunnen zijn.